packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -0,0 +1,216 @@
+   */
+  @Prop() public displayValuesAs: 'checkbox' | 'link' = 'checkbox';
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+
+    const options: NumericFacetOptions = {
+      facetId: this.facetId,
+      field: this.field,
+      numberOfValues: this.numberOfValues,
+      sortCriteria: this.sortCriteria,
+      rangeAlgorithm: this.rangeAlgorithm,
+      // TODO: implement manual ranges
+      generateAutomaticRanges: true,
+    };
+    this.facet = buildNumericFacet(this.bindings.engine, {options});
+    this.facetId = this.facet.state.facetId;
+    this.bindings.store.state.facets[this.facetId] = {
+      label: this.label,
+    };
+  }
+
+  private get numberOfSelectedValues() {
+    return this.facetState.values.filter(({state}) => state === 'selected')
+      .length;
+  }
+
+  private renderHeader() {
+    return (
+      <FacetHeader
+        i18n={this.bindings.i18n}
+        label={this.label}
+        onClearFilters={() => this.facet.deselectAll()}
+        numberOfSelectedValues={this.numberOfSelectedValues}
+        isCollapsed={this.isCollapsed}
+        onToggleCollapse={() => (this.isCollapsed = !this.isCollapsed)}
+      ></FacetHeader>
+    );
+  }
+
+  private renderValue(facetValue: NumericFacetValue, onClick: () => void) {
+    // TODO: implement formats
+    const displayValue = this.bindings.i18n.t('to', {
+      start: facetValue.start,
+      end: facetValue.end,
+    });
+    const isSelected = facetValue.state === 'selected';
+    switch (this.displayValuesAs) {
+      case 'checkbox':
+        return (
+          <FacetValueCheckbox
+            displayValue={displayValue}
+            numberOfResults={facetValue.numberOfResults}
+            isSelected={isSelected}
+            i18n={this.bindings.i18n}
+            onClick={onClick}
+          ></FacetValueCheckbox>
+        );
+      case 'link':
+        return (
+          <FacetValueLink
+            displayValue={displayValue}
+            numberOfResults={facetValue.numberOfResults}
+            isSelected={isSelected}
+            i18n={this.bindings.i18n}
+            onClick={onClick}
+          ></FacetValueLink>
+        );
+    }
+  }
+
+  private renderValuesContainer(children: VNode[]) {
+    return (
+      <ul part="values" class="mt-3">
+        {children}
+      </ul>
+    );
+  }
+
+  private renderValues() {
+    return this.renderValuesContainer(
+      this.facetState.values.map((value) =>
+        this.renderValue(value, () =>
+          this.displayValuesAs === 'link'
+            ? this.facet.toggleSingleSelect(value)
+            : this.facet.toggleSelect(value)
+        )
+      )
+    );
+  }
+
+  public render() {
+    if (this.searchStatusState.hasError) {
+      return;
+    }
+
+    if (!this.searchStatusState.firstSearchExecuted) {
+      return (
+        <FacetPlaceholder
+          numberOfValues={this.numberOfValues}
+        ></FacetPlaceholder>
+      );
+    }
```

Comment:
I don't understand why the rendering is not the same if the firstSearch has not been executed yet.

Reply:
Before any search is executed, we don't have any results or facet value, meanwhile we display a placeholder
![Screen Shot 2021-06-17 at 10 16 22 AM](https://user-images.githubusercontent.com/4923043/122414347-12d43900-cf55-11eb-8f77-e0518f36c607.png)